doc/API.md

```diff
@@ -711,253 +711,190 @@ The input byte array should be 80 bytes that form the solved block header. *Unli
+    "uploadspending":   "5678"  // hastings
+  }
 }
 ```
-'funds' is the number of hastings allocated for file contracts in the given
-period.
-
-'hosts' is the number of hosts that contracts will be formed with.
 
-'period' is the duration of contracts formed.
+#### /renter [POST]
 
-#### /renter/allowance [POST]
+modify settings that control the renter's behavior.
 
-Function: Sets the contract allowance.
-
-Parameters: none
+###### Query String Parameters [(with comments)](/doc/api/Renter.md#query-string-parameters)
 ```
-funds  types.Currency    (string)
-hosts  uint64
-period types.BlockHeight (uint64)
+funds  // (optional) hastings
+hosts  // (optional)
+period // (optional) block height
 ```
-'funds' is the number of hastings allocated for file contracts in the given
-period.
-
-'hosts' is the number of hosts that contracts will be formed with.
 
-'period' is the duration of contracts formed.
-
-Response: standard
-
-#### /renter/downloads [GET]
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
 
-Function: Lists all files in the download queue.
+#### /renter/contracts [GET]
 
-Parameters: none
+// TODO: is this description correct?
+returns all contracts that have been formed by the renter.
 
-Response:
-```
-struct {
-	downloads []struct {
-		siapath     string
-		destination string
-		filesize    uint64
-		received    uint64
-		starttime   Time (string)
-	}
+###### JSON Response [(with comments)](/doc/api/Renter.md#json-response-1)
+```javascript
+{
+  "contracts": [
+    {
+      "endheight":   50000, // block height
+      "id":          "1234567890abcdef0123456789abcdef0123456789abcdef0123456789abcdef",
+      "netaddress":  "12.34.56.78:9",
+      "renterfunds": "1234", // hastings
+      "size":        8192    // bytes TODO: is this the right unit?
+    }
+  ]
 }
 ```
-'siapath' is the siapath given to the file when it was uploaded.
-
-'destination' is the path that the file will be downloaded to.
-
-'filesize' is the size of the file being downloaded.
-
-'received' is the number of bytes downloaded thus far.
-
-'starttime' is the time at which the download was initiated.
 
-#### /renter/files
-
-Function: Lists the status of all files.
+#### /renter/downloads [GET]
 
-Parameters: none
+lists all files in the download queue.
 
-Response:
-```
-struct {
-	files []struct {
-		siapath        string
-		filesize       uint64
-		available      bool
-		renewing       bool
-		uploadprogress float64
-		expiration     types.BlockHeight (uint64)
-	}
+###### JSON Response [(with comments)](/doc/api/Renter.md#json-response-2)
+```javascript
+{
+  "downloads": [
+    {
+      "siapath":     "foo/bar.txt",
+      "destination": "/home/users/alice",
+      "filesize":    8192,                  // bytes
+      "received":    4096,                  // bytes
+      "starttime":   "2009-11-10T23:00:00Z" // RFC 3339 time TODO: why is this not a types.Timestamp?
```

Comment:
`types.Timestamp` is a bit unwieldy, since it's just a large number. `time.Time` is easier to process in other languages and is meant for human consumption. It's a pretty minor distinction though.